src/main/java/io/sniffy/servlet/BufferedServletResponseWrapper.java

```diff
@@ -88,8 +95,20 @@ public int getContentLength() {
 
     private String contentEncoding;
 
+    private boolean corsHeadersHeaderAdded = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/sniffy/sniffy/file/2287523352/issues/source?bid=3005960&fileBranchId=3190468#l98)